packages/modular-scripts/src/check/verifyModularRootPackageJson.ts

```diff
@@ -0,0 +1,56 @@
+import * as path from 'path';
+
+import * as fs from 'fs-extra';
+
+import getModularRoot from '../utils/getModularRoot';
+import { ModularPackageJson } from '../utils/isModularType';
+import * as logger from '../utils/logger';
+
+/**
+ * @param rootPackageJson package.json object from the modular root
+ * @returns Returns true if the packages directory is correctly included in `workspaces`, else false
+ */
+function checkWorkspaceHasPackages(rootPackageJson: ModularPackageJson) {
+  if (Array.isArray(rootPackageJson?.workspaces)) {
+    return !!rootPackageJson.workspaces.includes('packages/**');
+  }
+
+  return !!rootPackageJson?.workspaces?.packages?.includes('packages/**');
```

Comment:
Same here

Reply:
This one can evaluate to `undefined` though.